### ISSUE TITLE
unitTestFramework is not valid if unitTestRunner is None

### ISSUE THREAD

**Issue by smoore4moma** [unitejs/cli#5]
I am getting an error when I try to create a new project with the most basic set-up - no test, linters, etc.  Here is the command:

unite configure --packageName=unite-test --profile=AureliaJavaScript --linter=None --unitTestRunner=None --e2eTestRunner=None --cssPre=Css --cssPost=None --cssLinter=None --documenter=None

The error I am getting is "unitTestFramework is not valid if unitTestRunner is None."

Do I *need* to specify unit tests?  

Great idea and excited to try this out!

![img](https://user-images.githubusercontent.com/4989910/39631044-28ec308e-4f7f-11e8-8201-7c2ca1409a46.png)

Note:  similar error for e2e...

ERROR: e2eTestFramework is not valid if e2eTestRunner is None

**Comment by obany** [comment]
Thanks for reporting, this is because the profiles have a unitTestFramework and e2eTestFramework set by default and there is no way to turn them off.

You could use the following command line instead to achieve your same setup.

```shell
unite configure --packageName=unite-test --appFramework=Aurelia --moduleType=SystemJS --bundler=SystemJSBuilder --sourceLanguage=JavaScript --linter=None --unitTestRunner=None --e2eTestRunner=None --cssPre=Css --cssPost=None --cssLinter=None --documenter=None
```

I should probably set the unitTestFramework and e2eTestFramework to none internally if a profile specifies the runner and then they are overridden.

**Comment by smoore4moma** [comment]
Not a big deal.  It is helpful to have all of the modules anyway.  It just *seemed* like an option because I used the generator to set up my project.

**Comment by obany** [comment]
Fixed in 1.7.12
